doc/providers/csrf.rst

```diff
@@ -45,7 +45,7 @@ Service Provider are protected against CSRF by default.
 
 You can also use the CSRF protection even without using the Symfony Form
 component. If, for example, you're doing a DELETE action, you can check the
-CSRF token::
+CSRF token:
 
 .. code-block:: php
```

Comment:
Ok yes. Will fix it later today. I'll probably go through the 2.0 documentation first (I've seen a few typos and compatibility changes there). So I'll close this PR for now and create a new one later with all the changes.